packages/next-ui/Layout/components/LayoutHeaderContent.tsx

```diff
@@ -111,7 +111,7 @@ export function LayoutHeaderContent(props: LayoutHeaderContentProps) {
             gridTemplateAreas: `"left center right"`,
             gridTemplateColumns: '1fr auto 1fr',
             alignItems: 'center',
-            // columnGap: theme.spacings.xs,
+            gap: theme.page.horizontal,
```

Comment:
`theme.page.horizontal` is meant for the left/right side of the page. maybe do something like `theme.spacings.xs`?